libvalkeycluster/tests/test_utils.h

```diff
@@ -1,5 +1,5 @@
-#ifndef __TEST_UTILS_H__
```

Comment:
Why add VALKEY to each of these? Do you think there's a risk of macro name collission?

Reply:
Probably no collision in src/, I mostly followed how its done in the non-cluster headers.
```
src/async_private.h: #ifndef VALKEY_ASYNC_PRIVATE_H
src/fmacros.h:       #ifndef VALKEY_FMACRO_H
```
One (only?) benefit I can think of is that its already correct if we need to make a header public.